src/ansys/additive/core/parametric_study/parametric_study.py

```diff
@@ -555,6 +549,7 @@ def generate_single_bead_permutations(
                                     ColumnNames.BEAM_DIAMETER: d,
                                     ColumnNames.LASER_POWER: p,
                                     ColumnNames.SCAN_SPEED: v,
+                                    ColumnNames.PV_RATIO: p / v,
```

Comment:
Do we need a check for v != 0 here? Or is it handled elsewhere? Or should we ignore it and raise divide by zero exception?

Reply:
The value of the scan speed being in the mix max range will be validated by the `_validate_input` method.